src/DynamoCore/Graph/Nodes/CodeBlockUtils.cs

```diff
@@ -66,10 +66,6 @@ internal static string NormalizeLineBreaks(string text)
         /// <param name="statements">A list of Statement objects whose defined 
         /// variables are to be retrieved. This list can be empty but it cannot 
         /// be null.</param>
-        /// <param name="onlyTopLevel">Set this parameter to false to retrieve 
-        /// all variables defined in nested Statement objects.</param>
-        /// <returns>Returns a list of lists of variables defined by the given 
-        /// set of Statement objects.</returns>
```

Comment:
@Cristhian-Marin-asdk can you put back the <returns> section ?

Reply:
yes, I brought back the returns statement, except the comment for the onlyTopLevel parameter because is a parameter that no longer exists in the function.
![image](https://github.com/DynamoDS/Dynamo/assets/132492346/6db26902-9d89-46a8-a1ad-2218fb40dadd)